lib/event.rb

```diff
@@ -0,0 +1,18 @@
+class Event
+  attr_reader :name,
+              :crafts,
+              :attendees
+
+  def initialize(name, crafts, attendees)
+    @name          = name
+    @crafts        = crafts.first
```

Comment:
When we are initializing a new Event, we want to use all the crafts not only the first one
```suggestion
    @crafts        = crafts
```